src/logics/Logic.cc

```diff
@@ -65,7 +65,6 @@ Logic::Logic(opensmt::Logic_t _logicType) :
     , term_FALSE(mkConst(getSort_bool(), tk_false))
     , sym_TRUE(getSymRef(term_TRUE))
     , sym_FALSE(getSymRef(term_FALSE))
-    , sym_ANON(sym_store.newSymb(tk_anon, {}))
```

Comment:
Overall seem good, you've forgotten to remove tk_anon on line 40 though

Reply:
Good catch! Fixed now.